.github/workflows/build.yml

```
@@ -226,13 +222,24 @@ jobs:
           sudo mv /usr/bin/python3 /usr/bin/python3-default
           sudo ln -s ${{ env.pythonLocation }}/bin/python3 \
           /usr/bin/python3
+      - name: Process user data file
+        id: process-user-data
+        # Process the user data script with the windows admin password
+        # pulled from AWS Parameter Store. Return the admin password for
+        # Packer to consume.
+        run: |
+          output=$(python .github/scripts/process_user_data_file.py)
+          echo "::add-mask::$output"
+          echo "::set-output name=pass::$output"
       - name: Create machine image
         # This runs through the AMI creation process but does not
         # actually create an AMI
         run: |
+          PKR_VAR_admin_pass=\
```

Help out future readers who may not be familiar with this:
```suggestion
          # See https://www.packer.io/guides/hcl/variables#from-environment-variables
          # for details about PKR_VAR environment variables.
          PKR_VAR_admin_pass=\
```

---

resolved with [af4ef4c](https://github.com/cisagov/windows-commando-packer/pull/10/commits/af4ef4cc468b6b9df43d521520e4040d13ba90e2)

NOTE: I had to make the url shorter due to Pre-Commit failing due to that line being greater than 80 characters